scripts/release.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -l
+
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+IFS=$'\n\t'
+set -euxo pipefail
+
+# Make sure variables are set
+PRODUCTION=${PRODUCTION:-"false"}
+STAGING=${STAGING:-"false"}
+DRY_RUN=${DRY_RUN:-"true"}
+RELEASE_BUCKET=${RELEASE_BUCKET:-"false"}
+REPO_BRANCH=${REPO_BRANCH:-"master"}
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+git checkout $REPO_BRANCH
+
+# run the prepare script
+REFRESH_ASSETS=1 ./scripts/prepare.sh
+
+# build the buildpack
+./scripts/build.sh
+
+if [ "$DRY_RUN" == "true" ]; then
+  exit 0
+fi
+
+VERSION=$(cat VERSION)
```

Comment:
Can we run this before we exit if `DRY_RUN = true,` then output something like "Releasing version `$VERSION`" of the buildpack?